pkg/apis/extensions/validation/validation.go

```diff
@@ -447,6 +447,29 @@ func ValidateIngressUpdate(oldIngress, ingress *extensions.Ingress) errs.Validat
 	return allErrs
 }
 
+func ValidateIngressStatusUpdate(ingress, oldIngress *extensions.Ingress) errs.ValidationErrorList {
+	allErrs := errs.ValidationErrorList{}
+	allErrs = append(allErrs, apivalidation.ValidateObjectMetaUpdate(&ingress.ObjectMeta, &oldIngress.ObjectMeta).Prefix("metadata")...)
+
+	status := ingress.Status
+	for _, lbIngress := range status.LoadBalancer.Ingress {
+		if len(lbIngress.IP) > 0 {
```

Comment:
suggest splitting this out into ValidateLoadBalancerStatus (and using ingress as a prefix in the caller) so we can reuse for Service.Status.